orgs/organization.go

```diff
@@ -0,0 +1,144 @@
+package orgs
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"strings"
+	"time"
+
+	"github.com/LF-Engineering/dev-analytics-libraries/auth0"
+	"github.com/LF-Engineering/dev-analytics-libraries/elastic"
+	"github.com/LF-Engineering/dev-analytics-libraries/http"
+)
+
+// Org struct
+type Org struct {
+	OrgBaseURL       string
+	ESCacheURL       string
+	ESCacheUsername  string
+	ESCachePassword  string
+	AuthGrantType    string
+	AuthClientID     string
+	AuthClientSecret string
+	AuthAudience     string
+	AuthURL          string
+	Environment      string
+	httpClient       *http.ClientProvider
+	esClient         *elastic.ClientProvider
+	auth0Client      *auth0.ClientProvider
+}
+
+// SearchOrganization ...
+func (o *Org) SearchOrganization(name string, pageSize string, offset string) *SearchOrganizationResponse {
+	if name == "" {
+		log.Println("SearchOrganization: name is empty")
+		return nil
+	}
+	token, err := o.auth0Client.ValidateToken(o.Environment)
+	if err != nil {
+		log.Println(err)
+	}
+	headers := make(map[string]string, 0)
+	headers["Content-type"] = "application/json"
```

Comment:
the `httpClient.Request` method already adds a content type header. lets remove this

Reply:
done!